src/pages/OnboardingPersonalDetails/types.ts

```diff
@@ -10,6 +10,8 @@ type BaseOnboardingPersonalDetailsOnyxProps = {
 
     /** Saved onboarding admin chat report ID */
     onboardingAdminsChatReportID: OnyxEntry<string>;
+
+    onboardingPolicyID: OnyxEntry<string>;
```

Comment:
This is missing docs @nkdengineer can you please add some simple log

Reply:
@mountiny I updated.